stolostron/backlog#24526 rem clc apis

cadawson opened the pull request.

cadawson committed:
stolostron/backlog#24526 rem clc apis

cadawson committed:
stolostron/backlog#24526 rem add-on oper

zhiweiyin318 commented on release_notes/deprecate_remove.adoc:
```diff
@@ -61,54 +61,62 @@ Learn when parts of the product are deprecated or removed from {product-title}.
+|===
+| Product or category | Affected item | Version | Recommended action | More details and links
+
 | CertPolicyController
```
@cadawson  should we put CertPolicyController,ApplicationManager,IAMPolicyController,PolicyController,SearchCollector,WorkManager into removal part? because they are removed from 2.6 release.  Add-on operator is the same.

cadawson commented:
These are listed as removals. There is a section for removals, and a section for deprecations. Thanks!

zhiweiyin318 commented:
@cadawson I think we need to move these APIs CertPolicyController,ApplicationManager,IAMPolicyController,PolicyController,SearchCollector,WorkManager into removal section too.

cadawson commented:
Hi @zhiweiyin318 . I'm not sure I understand. There is a table for API deprecations, and a table for API removals. I added the removed APIs to the API removal table. Since the operator wasn't an API, I added it to the general removal table. 

From what I am seeing, the APIs that you mention are already in the removal table for the APIs. Are you seeing something different? Thanks!

zhiweiyin318 commented:
@cadawson I'm so sorry, my bad. the github compare page shows the changes in removal table with the original  deprecations table.  the PR is good. 
/lgtm

cadawson commented:
Thanks @zhiweiyin318 ! I have seen some times where things appeared a little strangely in the Git preview. Glad that it is working now, and looks ok.

oafischer reviewed (approved):
Thanks, looks good to me!
